apache/trafficcontrol
# Do not include Scenes dev dependencies in Traffic Stats

NPM dev dependencies were being included in Traffic Stats. #8010 ensures that dev dependencies are omitted.
<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Stats
- Build system 

## What is the best way to verify this PR?

1. build Traffic Stats RPM
2. Look at the files
3. Ensure it does not include packages like `e2e` in the `node_modules` folder

## If this is a bugfix, which Traffic Control versions contained the bug?

8.0.1

## PR submission checklist
